positron/modules/process.js

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const Cc = Components.classes;
+const Ci = Components.interfaces;
+const Cr = Components.results;
```

Comment:
> nit: defined but never used.

I've replaced these definitions with the destructuring assignment variant you suggested (again, over in #2, where the code was moved to this location) in 86952e2.

We could automatically inject these globals into all the modules that get Components, so the modules don't have to define them themselves.

> We should probably enable automatic linting of these files as part of some CI process.

Indeed!